lexer/lexer_test.go

```diff
@@ -42,6 +42,7 @@ j--
 /*/*/
 /* This is a
    multiline comment */
+..
```

Comment:
Please add something about

1..2
1 .. 3
1 .. 4
1.. 5

To make sure how the lexer reacts with things that could be interpreted as number

And maybe

1.

Reply:
there are no range yet (1. is there already in many places)

1..2 is actually
```
1.
.2
```
but it's always been that

1 .. 2

complains about .. being undefined (unless used inside a varargs or set before)

short version is we need #56 and more documentation about the language

Reply:
fuzz testing would be nice too

Reply:
#101 for fancier fuzz test

Reply:
> there are no range yet

Not yet,y point was to add these to tests and see what happens.

So yes,the fuzzy testing might help